src/main/java/io/moderne/devcenter/DependencyVulnerabilityCheck.java

```diff
@@ -0,0 +1,189 @@
+
+@Value
+@EqualsAndHashCode(callSuper = false)
+public class DependencyVulnerabilityCheck extends UpgradeMigrationCard {
+
+    @Option(displayName = "Fixable only",
+            description = "Only report vulnerabilities that have a known fix available. By default this is true.",
+            required = false)
+    @Nullable
+    Boolean fixableOnly;
+
+    @Option(displayName = "Add transitive",
+            description = "Whether to include transitive dependencies in the vulnerability check. By default this is true.",
+            required = false)
+    @Nullable
+    Boolean addTransitive;
+
+    @Option(displayName = "Measure",
+            description = "Choose to report Severity or UpgradeDelta.",
+            valid = {"Severity", "UpgradeDelta"})
+    String measure;
+
+    @Override
+    public String getDisplayName() {
+        return "Vulnerabilities status";
+    }
+
+    @Override
+    public String getInstanceName() {
+        return "Vulnerable dependencies";
+    }
+
+    @Override
+    public String getDescription() {
+        return "Determine the current state of a repository relative to its vulnerabilities.";
+    }
+
+    @Override
+    public TreeVisitor<?, ExecutionContext> getVisitor() {
+        org.openrewrite.java.dependencies.DependencyVulnerabilityCheck scan = new org.openrewrite.java.dependencies.DependencyVulnerabilityCheck("runtime", addTransitive == null || addTransitive, UpgradeDelta.major);
+        AtomicReference<DependencyVulnerabilityCheckBase.@Nullable Accumulator> accumulator = new AtomicReference<>();
+        return Preconditions.check(Preconditions.or(new IsBuildGradle<>(), new FindMavenProject().getVisitor()), new TreeVisitor<Tree, ExecutionContext>() {
+            @Override
+            public Tree preVisit(Tree tree, ExecutionContext ctx) {
+                stopAfterPreVisit();
+                if (accumulator.get() == null) {
+                    accumulator.set(scan.getInitialValue(ctx));
+                }
+                scan.getScanner(accumulator.get()).visitNonNull(tree, ctx);
+
+                DataTableRowWatcher<VulnerabilityReport.Row> dataTableWatcher = new DataTableRowWatcher<>(scan.getVulnerabilityReport(), ctx);
+                dataTableWatcher.start();
+                scan.generate(accumulator.get(), ctx);
+
+                List<VulnerabilityReport.Row> vulnerableDependencies = dataTableWatcher.stop();
+
+                if (vulnerableDependencies.isEmpty()) {
+                    upgradesAndMigrations.insertRow(ctx, DependencyVulnerabilityCheck.this, UpgradeDeltaMeasure.Completed, "No vulnerabilities found");
+                }
+
+                for (VulnerabilityReport.Row row : vulnerableDependencies) {
+                    if ((fixableOnly == null || fixableOnly) && row.getFixedVersion() == null) {
+                        upgradesAndMigrations.insertRow(ctx, DependencyVulnerabilityCheck.this, UpgradeDeltaMeasure.Completed, "Only vulnerabilities are not fixable.");
```

Comment:
The phrasing here at the end is a bit odd; how about something like:
```suggestion
                        upgradesAndMigrations.insertRow(ctx, DependencyVulnerabilityCheck.this, UpgradeDeltaMeasure.Completed, "No fix available for vulnerability.");
```